src/content/it/2020/resource-hints.md

```diff
@@ -0,0 +1,366 @@
+discuss: 2057
+results: https://docs.google.com/spreadsheets/d/1lXjd8ogB7kYfG09eUdGYXUlrMjs4mq1Z7nNldQnvkVA/
+queries: 21_Resource_Hints
+featured_quote: During the past year resource hints increased in adoption, and they have become essential APIs for developers to have more granular control over many aspects of resource prioritizations and ultimately, user experience.
+featured_stat_1: 33%
+featured_stat_label_1: Sites using <code>dns-prefetch</code>
+featured_stat_2: 9%
+featured_stat_label_2: Sites using <code>preload</code>
+featured_stat_3: 4.02%
+featured_stat_label_3: Sites using native lazy loading
+---
+
+## Introduzione
+
+Negli ultimi dieci anni [resource hints](https://www.w3.org/TR/resource-hints/) sono diventati essenzialmente delle primitive che consentono agli sviluppatori di migliorare le prestazioni della pagina e l'user experience.
+
+Il precaricamento delle risorse e il fatto che i browser applichino una priorità intelligente è qualcosa che è stato effettivamente avviato nel lontano 2009 da IE8 con qualcosa chiamato [preloader](https://speedcurve.com/blog/load-scripts-async/). Oltre al parser HTML, IE8 aveva un preloader leggero che analizzava i tag in grado di avviare richieste di rete (`<script>`, `<link>` e `<img>`).
+
+Negli anni successivi, i fornitori di browser hanno svolto sempre più il grosso del lavoro, ognuno aggiungendo la propria salsa speciale su come dare la priorità alle risorse. Ma è importante capire che da solo il browser ha alcune limitazioni. In qualità di sviluppatori, tuttavia, possiamo superare questi limiti facendo un buon uso delle resource hints e aiutandoci a decidere come assegnare la priorità alle risorse, determinando quali dovrebbero essere fetched o preprocessed per aumentare ulteriormente le prestazioni della pagina.
+
+In particolare possiamo citare alcune vittorie raggiunte/ottenute nell'ultimo anno riguardante le resource hints:
+- [CSS-Tricks](https://www.zachleat.com/web/css-tricks-web-fonts/) web fonts vengono visualizzati più velocemente su un primo rendering 3G.
+- [Wix.com](https://www.youtube.com/watch?v=4QqlGgF8Y2I&t=1469) l'utilizzo delle resource hints ha ottenuto un miglioramento del 10% per FCP.
+- [Ironmongerydirect.co.uk](https://andydavies.me/blog/2019/03/22/improving-perceived-performance-with-a-link-rel-equals-preconnect-http-header/) ha utilizzato la preconnect per migliorare il caricamento dell'immagine del prodotto di 400 ms alla mediana e di oltre 1s al 95° percentile.
+- [Facebook.com](https://engineering.fb.com/2020/05/08/web/facebook-redesign/) preload utilizzato per una navigazione più veloce.
+
+Diamo un'occhiata alle resource hints più predominanti supportati ultimamente dalla maggior parte dei browser: `dns-prefetch`, `preconnect`, `preload`, `prefetch`, e native lazy loading.
```

Comment:
Let's see if we try to translate "Native Lazy Loading" it will be "Nativo Caricamento Pigro", as Italian it kind of makes me think Winnie The Pooh sleeping under a tree. 😅
I think is good keeping this word in english, generally many coding term are better in english in Italy.